scripts/bash/downloadEnvironmentVariables.sh

```
@@ -27,12 +29,28 @@ if [[ ! $(bw login --check) ]]; then
 
+cleanup() {
+    echo
+    echo -e "${BLUE}==>️${RESET} ${BOLD}Logging out of Bitwarden${RESET}"
+    bw logout
+
+    # Clean up detritus on macOS
+    # macOS and Ubuntu’s interfaces for sed are slightly different. In this script, we use it in a
+    # way that’s compatible to both (by not supplying a suffix for the -i flag), but this causes
+    # macOS to generate backup files which we don’t need.
+    if [[ $(uname) = 'Darwin' ]]; then
+        rm -f "$script_dir"/../../env/*.env-e "$script_dir"/../../packages/*/.env-e
```

Shellcheck (0.10.0)

[warning] script_dir is referenced but not assigned.

(SC2154)